src/element/mod.rs

```diff
@@ -256,13 +256,16 @@ impl IonEq for Element {
 /// An `(annotations, value)` pair representing an Ion value.
 #[derive(Debug, Clone)]
 pub struct Element {
-    annotations: Vec<Symbol>,
+    annotations: Annotations,
     value: Value,
 }
 
 impl Element {
-    pub fn new(annotations: Vec<Symbol>, value: Value) -> Self {
-        Self { annotations, value }
+    pub(crate) fn new(annotations: Annotations, value: impl Into<Value>) -> Self {
```

Comment:
Great question. For now it isn't because it's not because there isn't an efficient implementation of `From<Vec<Symbol>> for Annotations`. This is due to use of a blanket trait impl for an `IntoIterator<S: Into<Symbol>>` that allows us to accept slices, arrays, vecs, and other containers of `&str`, `String`, `Symbol`, etc. I have a detailed comment on the existing trait impl.

Requiring an `Annotations` means that callers will need to build it themselves using `pub(crate) fn Annotations::new()`, which takes a `Vec<Symbol>`.